workflow/schemas/config.schema.yaml

```diff
@@ -33,22 +34,18 @@ properties:
       - artifacts
       - background
 
-
-  filter_vcf_on_format:
+  add_multi_snv_in_codon:
     type: object
     properties:
       benchmark_repeats:
         type: integer
-        description: set number of times benchmark should be repeated for filter_vcf_on_format
+        description: set number of times benchmark should be repeated for add_multi_snv_in_codon
       container:
         type: string
         description: name or path to docker/singularity container
-      filter:
-        type: string
-        description: filtering parameters to filter_vcf_on_format
-      extra:
+      use_filter:
         type: string
-        description: extra parameters to filter_vcf_on_format
+        description: When 'True' only those position with filter 'PASS' are considered
```

Comment:
There is a boolean type available which is `False` by default.

Reply:
This has nothing to do with my pull request. Just changed the order to alphabetical. 
But it is a good point and I will change it in another pull request.

Reply:
Did not notice it before, hence, the comment.